### PR TITLE
Add conda build for mosdef_cassandra

### PR DESCRIPTION
**Pull request by rsdefever**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rsdefever** [comment]
@conda-forge/help-python

Good afternoon, I'm having some trouble getting my build to work due to dependency conflicts. 

I'm trying to build for linux and osx, Python 3.x.

If I use a `noarch` build, and `python >=3.6`, I am able to get the linux and osx builds to be successful. However, the `linter` doesn't like me adding `skip: True  # [win]` selector. 

If I do not use `noarch`, and use the `skip: True  # [win]` selector and `skip: True  # [py<35]` selector, then the linux and osx builds fail.

I'm a little confused and stuck on how to proceed. Thanks in advance!

File: recipes/mosdef_cassandra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/MaginnGroup/mosdef_cassandra/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<35]
```
Comment (bdice) [inline recipes/mosdef_cassandra/meta.yaml]: You might be overriding the first `skip` when you define the second `skip`. Try:
```suggestion
  skip: True  # [win or py<35]
```

File: recipes/mosdef_cassandra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```
Comment (bdice) [inline recipes/mosdef_cassandra/meta.yaml]: This doesn't quite match the requirements listed here: https://github.com/MaginnGroup/mosdef_cassandra/blob/master/requirements.txt?

**Comment by isuruf** [comment]
@rsdefever, for noarch: python packages, windows failure is okay if a dependency is missing.

**Comment by rsdefever** [comment]
@isuruf Thanks for the information! The above should be good to go then?

**Comment by rsdefever** [comment]
@conda-forge/staged-recipes I think this is ready for review! Thanks!

**Pull request merged**